.gitlab-ci.yml

```diff
@@ -590,7 +590,7 @@ dogstatsd_rpm-x64:
 # build Dogstastd package for rpm-x64
 dogstatsd_suse-x64:
   stage: package_build
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/suse_x64:latest
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/suse_x64:v1585177-e18cf5b
```

Comment:
I think this should go back to `latest` yes?  And when `datadog-agent-builders` runs and promotes to latest it will get the right thing?